code/web/src/modules/survey/api/actions.js

```diff
@@ -40,5 +43,32 @@ export const moveBackward = () => (
   }
 )
 
+// export const getProducts = () => (
+//   dispatch => {
+//     dispatch({
+//       type: GET_PRODUCTS
+//       products
+//     })
+//   }
+// )
 
+// export const getImages = () => (
+//   dispatch => {
+//     return axios.get('https://randomfox.ca/images/')
+//       .then(response => {
+//         dispatch({
+//           type: GET_IMAGES,
+//           item: response.data
+//         })
+//       })
+//       .catch(error => console.log(error))
+//   }
+// )
```

Comment:
This is the endpoint request yes? Hopefully we get our side of this more complete for you guys very soon

Reply:
Yeah, the endpoint will go in the place of the randomfox link! Appreciate y'all